biztalk/core/office365-adapters.md

```diff
@@ -17,42 +17,35 @@ manager: dougeby
 # Office 365 Outlook adapters in BizTalk
 
 ## Overview
-**Starting with [!INCLUDE[bts2016_md](../includes/bts2016-md.md)] Feature Pack 3**, you can send and receive messages between BizTalk Server and Office 365 Outlook features. The following adapters are included in Feature Pack 3:
 
-- [Office 365 Outlook Email adapter](office365-mail-adapter.md)
-- [Office 365 Outlook Calendar adapter](office365-calendar-adapter.md)
-- [Office 365 Outlook Contact adapter](office365-contact-adapter.md)
+Use the Office 365 Outlook adapters in BizTalk to send and receive messages between BizTalk Server and Office 365 Outlook features:
+
+- [Office 365 Outlook Email adapter](../core/office365-mail-adapter.md)
+- [Office 365 Outlook Calendar adapter](../core/office365-calendar-adapter.md)
+- [Office 365 Outlook Contact adapter](../core/office365-contact-adapter.md)
 
 ## Prerequisites
 
-* Have an [Office 365 account](https://outlook.office365.com)
-* Install [Feature Pack 3](https://aka.ms/bts2016fp3) on your BizTalk Server
-* Use an account that is a member of the SSO Administrators group
+- Have an [Office 365 account](https://outlook.office365.com)
+- [Configure BizTalk TMS service](../core/configure-biztalk-server.md#configure-biztalk-tms)
 
 ## TMS overview
 
-BizTalk Server TMS is a service that refreshes the Office 365 OAuth tokens used by BizTalk. It refreshes these tokens periodically, ensuring that the tokens always remain valid. It has a dependency on Enterprise Single Sign On service (ENT SSO), and must be installed on a computer that hosts the master secret server. 
-
-## Install BizTalk Server TMS
-
-1. Install [Feature Pack 3](https://aka.ms/bts2016fp3).
-2. In `\Program Files (x86)\Microsoft BizTalk Server <your version>`, run BizTalkTMS.msi.
-3. Enter the username and password of a user that's a member of the SSO Administrators group. 
-
-![BizTalk Server TMS setup](../core/media/BizTalk-TMS.png)
+BizTalk TMS is a service that refreshes the Office 365 OAuth tokens used by BizTalk. It refreshes these tokens periodically, ensuring that the tokens always remain valid.

```

Comment:
Should we use "manage' instead of 'refresh'
Especially given the future plans around TMS.